libs/api/domains/consultation-portal/src/lib/models/adviceResult.model.ts

```diff
@@ -0,0 +1,26 @@
+import { ObjectType, Field } from '@nestjs/graphql'
+import { DocumentInfoResult } from './documentInfoResult.model'
+
+@ObjectType()
```

Comment:
Need to namespace the name here and in all other object and input types. Something like `@ObjectType('ConsultationPortalAdviceResult')`